lib/timber/events/http.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Timber.Events.HTTP do
+  @moduledoc """
+  Utility module for common HTTP related events.
+  """
+
+  def normalize_method(method) do
```

Comment:
True, I don't like it there. I'll move them. I was trying to avoid dumping everything in Utils and keeping it a little more organized, but it's not that large yet.